docs/Other SDL Features/Remote Control Vehicle Features/index.md

```diff
@@ -390,7 +390,7 @@ After you subscribe to the `InteriorVehicleDataNotification` you must also subsc
 
 ###### RPC < v6.0
 ```java
-GetInteriorVehicleData getInteriorVehicleData = new GetInteriorVehicleData(ModuleType.CLIMATE);
+GetInteriorVehicleData getInteriorVehicleData = new GetInteriorVehicleData(ModuleType.CLIMATE).setSubscribe(true);
```

Comment:
```suggestion
GetInteriorVehicleData getInteriorVehicleData = new GetInteriorVehicleData(ModuleType.CLIMATE)
.setSubscribe(true);
```

Reply:
Maybe it would look better like this? It also matches how you have it for RPC v6.0+

Reply:
I see what you're saying. I agree and have made the change. Thanks.